extensions/amp-sidebar/1.0/amp-sidebar-1.0.md

```diff
@@ -58,103 +58,232 @@ limitations under the License.
-- The current state of the sidebar is exposed via the `open` attribute that is set on the `amp-sidebar` tag when the side bar is open on the page.
+  <nav toolbar="(min-width: 0px)" toolbar-only>
+    <ul>
+      <li>
+        <input placeholder="Search..."/>
+      </li>
+    </ul>
+  </nav>
+</amp-sidebar>
+```
 
-## Actions
-The `amp-sidebar` exposes the following actions you can use [AMP on-syntax to trigger](../../../spec/amp-actions-and-events.md):
+Example (toolbar-only on individual `<li>` elements):
 
-<table>
-  <tr>
-    <th>Action</th>
-    <th>Description</th>
-  </tr>
-  <tr>
-    <td>open (default)</td>
-    <td>Opens the sidebar</td>
-  </tr>
-  <tr>
-    <td>close</td>
-    <td>Closes the sidebar</td>
-  </tr>
-  <tr>
-    <td>toggle</td>
-    <td>Toggles the sidebar state</td>
-  </tr>
-</table>
+```html
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
+  <ul>
+    <li> Nav item 1</li>
+    <li> Nav item 2</li>
+    <li> Nav item 3</li>
+    <li> Nav item 4</li>
+    <li> Nav item 5</li>
+    <li> Nav item 6</li>
+  </ul>
 
+  <nav toolbar="(min-width: 768px) and (max-width: 1024px)">
+    <ul>
+      <li>
+        Publisher Logo
+      </li>
+      <li>
+        <input placeholder="Search..." toolbar-only/>
+      </li>
+    </ul>
+  </nav>
+</amp-sidebar>
+```
 
-### Examples
+Example (target attribute):
 
 ```html
-<button on="tap:sidebar.open"> = </button>
-<amp-sidebar id="sidebar" layout="nodisplay">
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
   <ul>
-    <li on="tap:sidebar.toggle">Toggle</li>
-    <li on="tap:sidebar.close">Close</li>
+    <li> Nav item 1</li>
+    <li> Nav item 2</li>
+    <li> Nav item 3</li>
+    <li> Nav item 4</li>
+    <li> Nav item 5</li>
+    <li> Nav item 6</li>
   </ul>
+
+  <nav toolbar="(max-width: 767px)" target="toolbar-target">
+    <ul>
+      <li>
+        <input placeholder="Search..."/>
+      </li>
+    </ul>
+  </nav>
 </amp-sidebar>
+
+<div id="toolbar-target">
+</div>
 ```
 
+
+{% call callout('Tip', type='success') %}
+See live demos at [AMP By Example](https://ampbyexample.com/components/amp-sidebar/).
+{% endcall %}
+
+## Attributes
+
+##### side
+
+Indicates what side of the page the sidebar should open from, either `left` or `right`.  If a `side` is not specified, the `side` value will be inherited from the `body` tag's `dir` attribute (`ltr` => `left` , `rtl` => `right`); if no `dir` exists, the `side` defaults to `left`.
+
+##### layout
+
+Specifies the display layout of the sidebar, which must be `nodisplay`.
+
+##### open
+
+This attribute is present when the sidebar is open.
+
+
+##### data-close-button-aria-label**
+
+Optional attribute used to set ARIA label for the close button added for accessibility.
+
+
+##### toolbar
+
+This attribute is present on child `<nav toolbar="(media-query)">` elements, and accepts a media query of when to show a toolbar.
```

Comment:
Add this sentence to the three new attributes: `See the [Toolbar](#toolbar) section for more information on using toolbars`.